### ISSUE TITLE
Create importer tool for Tim Hortons

### ISSUE THREAD

**Issue by joshi1983** [hhaccessibility/hhaccessibility.github.io#392]
Tim Hortons has a lot of locations and for each location, there are items like 'wifi' which correspond with questions from our criteria checklist.

Create a tool using Python that can download location information for a given area and convert that information into CSV format. The more columns of information the better but we'll be very interested in having a minimum of longitude, latitude, name, address, and wifi.

This new importer tool should be saved to the importers/timhortons.com directory.  Review our existing import tools to see what packages and code structure they use and be consistent.

Here is an example of a location search that we'd expect the tool to collect information from:
http://www.timhortons.com/ca/en/locations/locations.php?q_loc=n8x3e3&q_rad=10

That URL contains the center postal code(n8x3e3) and a search radius of 10km.  If you can make a CSV with all the information in that page programmatically, this task is essentially complete.

Here is a screenshot from the above location search page:
![image](https://user-images.githubusercontent.com/11838191/31120527-c3097a34-a802-11e7-9cdc-8f2d8a7bc4f1.png)

**Comment by joshi1983** [comment]
Shaumik Baki responded to one of my kijiji ads and may work on this.  Shaumik, comment here when you're ready to start the work.

**Comment by ShaumikB** [comment]
I'll begin learning for this. I'll let you know if I run into any roadblocks.

**Comment by joshi1983** [comment]
Great, Shaumik.

**Comment by yepterence** [comment]
Josh, let me know where you'd like me to help out.

**Comment by joshi1983** [comment]
Shaumik, how is this going?  Hit any roadblocks while trying to make the Tim Hortons import tool?

Terence was tempted to take this issue but I encouraged him to make a tool for McDonalds instead.

**Comment by joshi1983** [comment]
I merged a new Tim Hortons import tool.  It doesn't include any accessibility information, unfortunately.  Between the time the issue was made and now, the features that once showed if WIFI was supported and indoor seating was removed from timhortons.com and I wasn't able to find the information elsewhere on the site.

I'm closing this issue instead of leaving it open for testing since there is no functionality that is easily tested by a non-developer.
